src/db/schemas/notesSchema.js

```diff
@@ -0,0 +1,11 @@
+import mongoose from "mongoose";
+
```

Comment:
Move please this folder and file to the src/db, because it's referenced to the DB